open-metadata-implementation/admin-services/admin-services-spring/src/main/java/org/odpi/openmetadata/adminservices/spring/ConfigResource.java

```diff
@@ -17,7 +18,7 @@
  * configuration document is not found, a new one is created.
  */
 @RestController
-@RequestMapping("/open-metadata/admin-services/users/{userId}/servers/{serverName}")
```

Comment:
Better to put the new method in a new Resource file rather than having to change all of the existing methods

Reply:
changed